api/python/provisioner/commands/auto_deploy.py

```diff
@@ -55,19 +52,23 @@ class AutoDeploy(SetupCmdBase, CommandParserFillerMixin):
 
     @staticmethod
     def deployment_validations(deploy_check):
-        checker = Check()
+        check_import = importlib.reload(check)
+
+        check_cmd = getattr(check_import, 'Check')
```

Comment:
I think, we can just use here:
```
check = importlib.reload(check)

check_cmd = check.Check
```
Or even more change ` precheck_dm().make_decision(check_result=check_res)` to
```python
check.PreChecksDecisionMaker().make_decision(check_result=check_res)
```

Reply:
There is also this option
```python
check = importlib.reload(check)

PreChecksDecisionMaker = check.PreChecksDecisionMaker
```
and use it as ordinary
```python
PreChecksDecisionMaker().make_decision(check_result=check_res)
```

Reply:
Comment addressed. Thanks.